src/components/StoreInfo/StoreSocialTab.tsx

```diff
@@ -0,0 +1,146 @@
+            </SocialContainer>
+
+            <SocialContainer>
+                <TwitterShareButton url={location} className="share">
+                    <SocialIcon
+                        network="twitter"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>TWITTER</SocialTextLabel>
+                </TwitterShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <EmailShareButton url={location} className="share">
+                    <SocialIcon
+                        network="email"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>EMAIL</SocialTextLabel>
+                </EmailShareButton>
+            </SocialContainer>
+
+            <SocialContainer>
+                <ClipboardContainer ref={target}>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <ClipboardIcon
+                            size={28}
+                            style={clipboardStyle}
+                        />
+                    </CopyToClipboard>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <SocialTextLabel>COPY TO CLIPBOARD</SocialTextLabel>
+                    </CopyToClipboard>
+                    <Overlay target={target.current} show={show} placement="bottom">
+                        {(props) => (
+                            <Tooltip id="overlay-example" {...props} style={{
+                                backgroundColor: socialIconForegroundColor,
+                                color: socialIconBackgroundColor,
+                                padding: '5px 15px 5px 15px',
+                                borderRadius: 3,
+                                ...props.style,
+                            }}>
+                                COPIED!
+                            </Tooltip>
+                        )}
+                    </Overlay>
+                </ClipboardContainer>
+            </SocialContainer>
+        </React.Fragment>
+    );
+};
+
+const HeaderText = styled.div`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 22px;
+    color: #1E1E1E;
+    margin: -10px 0px 30px 10px;
+`;
+
+const SocialTextLabel = styled.label`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 22px;
+    align-items: center;
+    letter-spacing: 0.15em;
+    text-transform: uppercase;
+    color: #000000;
+    padding-left: 10px;
+`;
+
+const SocialContainer = styled.div`
```

Comment:
Could you add cursor: pointer here? Or does default already handle it

Reply:
Done